datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -494,9 +494,17 @@ def monotonic_count(self, name, value, tags=None, hostname=None, device_name=Non
         - **device_name** (_str_) - **deprecated** add a tag in the form `device:<device_name>` to the `tags`
             list instead.
         - **raw** (_bool_) - whether to ignore any defined namespace prefix
+        - **flush_first_value** (_bool_) - whether to sample the first value
```

Comment:
If `flush_first_value` is set to `True`, does that mean you _don't_ sample the first value?

Reply:
No the opposite